From 2eb6d1b761a3fddea28294959bde7e9c7ef88043 Mon Sep 17 00:00:00 2001
Message-Id: <2eb6d1b761a3fddea28294959bde7e9c7ef88043.1319086508.git.acho@suse.com>
From: Cho, Yu-Chen <acho@suse.com>
Date: Thu, 20 Oct 2011 12:52:02 +0800
Subject: [PATCH 1/2] add device entry for Broadcom Valentine combo card
Patch-mainline: Submitted 3.2-rc1
References: bnc#726176

This device declares itself to be vendor specific
It therefore needs to be added to the device table
to make btusb bind.

Signed-off-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/bluetooth/btusb.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/drivers/bluetooth/btusb.c b/drivers/bluetooth/btusb.c
index abfc4ee..5893ba4 100644
--- a/drivers/bluetooth/btusb.c
+++ b/drivers/bluetooth/btusb.c
@@ -97,6 +97,10 @@ static struct usb_device_id btusb_table[] = {
 	/* Ericsson with non-standard id */
 	{ USB_DEVICE(0x0bdb, 0x1002) },
 
+	/* Broadcom 20702 Bluetooth 4.0 Adapter */
+	/* Broadcom Valentine 802.11bgn 1x1 Combo HMC */
+	{ USB_DEVICE(0xa5c, 0x21e3) },
+
 	/* Canyon CN-BTU1 with HID interfaces */
 	{ USB_DEVICE(0x0c10, 0x0000) },
 
-- 
1.7.3.4

